server/helpers/adapters_tracker.go

```diff
@@ -119,11 +124,90 @@ func (a *AdaptersTracker) DeployAdapter(ctx context.Context, adapter models.Adap
 			return ErrAdapterAdministration(err)
 		}
 
+	case "kubernetes":
+		kubeClient, err := meshkitkube.New([]byte(""))
+		if err != nil {
+			return ErrAdapterAdministration(err)
+		}
+
+		port, _ := strconv.Atoi(adapter.Location)
+
+		// Create a deployment
+		deployment := &appsv1.Deployment{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      adapter.Name,
+				Namespace: core.MesheryNamespace,
+			},
+			Spec: appsv1.DeploymentSpec{
+				Selector: &metav1.LabelSelector{
+					MatchLabels: map[string]string{
+						"app": adapter.Name,
+					},
+				},
+				Replicas: int32Ptr(1),
+				Template: corev1.PodTemplateSpec{
+					ObjectMeta: metav1.ObjectMeta{
+						Labels: map[string]string{
+							"app": adapter.Name,
+						},
+					},
+					Spec: corev1.PodSpec{
+						Containers: []corev1.Container{
+							{
+								Name:  adapter.Name,
+								Image: "layer5/" + adapter.Name + ":stable-latest",
```

Comment:
@Azanul is this same stable-latest assumption already being made in the current deployments?

Reply:
@leecalcote No, currently the version is being inferred from the config file by meshery version

Reply:
@Azanul that is fantastic. 💯

Reply:
The version here is hardcoded because meshery server doesn't have that config file.